instrumentation-security/servlet-2.4/src/main/java/javax/servlet/ServletContainerInitializer_Instrumentation.java

```diff
@@ -0,0 +1,26 @@
+package javax.servlet;
+
+import com.newrelic.api.agent.weaver.MatchType;
+import com.newrelic.api.agent.weaver.Weave;
+import com.newrelic.api.agent.weaver.Weaver;
+import com.nr.instrumentation.security.servlet24.HttpServletHelper;
+
+import java.util.Set;
+
+@Weave(type = MatchType.Interface, originalName = "javax.servlet.ServletContainerInitializer")
+public class ServletContainerInitializer_Instrumentation {
+    public void onStartup(Set<Class<?>> c, ServletContext ctx) throws ServletException {
+        try {
+            Weaver.callOriginal();
+        } finally {
+            postProcessing(ctx);
+        }
+    }
+
+    private static void postProcessing(ServletContext context) {
+        try {
```

Comment:
No need of try catch as inner method already contains it

Reply:
Fixed this in commit [8652436](https://github.com/newrelic/csec-java-agent/pull/52/commits/86524360e0e9d71dc5696d7da747931d01b310df)